rileylink/src/main/java/info/nightscout/androidaps/plugins/pump/common/hw/rileylink/Platform.java

```diff
@@ -0,0 +1,66 @@
+package info.nightscout.androidaps.plugins.pump.common.hw.rileylink;
+
+import android.os.Build;
+import android.os.Handler;
+import android.os.Looper;
+
+import java.util.concurrent.Executor;
+import java.util.concurrent.Executors;
+
+public class Platform {
+    private static final Platform PLATFORM = findPlatform();
+    Handler mainhandler = new Handler(Looper.getMainLooper());
+
+    public static Platform get() {
+        return PLATFORM;
+    }
+
+
+    private static Platform findPlatform() {
+        try {
+            Class.forName("android.os.Build");
+            if (Build.VERSION.SDK_INT != 0) {
+                return new Android();
+            }
+        } catch (ClassNotFoundException ignored) {
+        }
+        return new Platform();
+    }
```

Comment:
This seems to be code to detect the platform while we know we are running on Android?